apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -29,7 +29,7 @@ import { useDispatch } from "react-redux";
 import { Grid, Icon, Modal } from "semantic-ui-react";
 import { RolePermissions } from "./user-role-permissions";
 import { AddUserWizardSummary } from "./wizard-summary";
-import { SCIMConfigs } from "../../../../extensions/configs";
+import { SCIMConfigs } from "../../../../extensions/configs/scim"; //keep statement as this to avoid cyclic dependency
```

Comment:
```suggestion
// Keep statement as this to avoid cyclic dependency. Do not import from config index.
import { SCIMConfigs } from "../../../../extensions/configs/scim";
```